text/0000-trait-alias.md

```diff
@@ -133,23 +133,27 @@ parameter declarations*.
 
 When using a trait alias as an object type, it is subject to object safety restrictions _after_ substituting the aliased traits. This means:
 
-1. It contains an object safe trait and zero or more of these other bounds: `Send`, `Sync`, `'static` (that is, `trait Show = Display + Debug;` would not be object safe).
+1. It contains an object safe trait, optionally a lifetime, and zero or more of these other bounds: `Send`, `Sync` (that is, `trait Show = Display + Debug;` would not be object safe).
```

Comment:
Oh, should also specify what happens if I have `trait Foo = Bar + 'a;` and I use `Foo + 'b` as a trait object.
It has to either be an error or require that `'b` outlive `'a` (i.e. you can have `Foo + 'static` but not shorter than `'a`).